app/src/interfaces/_system/system-token/system-token.vue

```diff
@@ -62,6 +65,14 @@ function emitValue(newValue: string | null) {
 	emit('input', newValue);
 	localValue.value = newValue;
 }
+
+function select(event: FocusEvent) {
+	if (localValue.value) (event.target as HTMLInputElement | null)?.select();
+}
+
+function deselect() {
+	window.getSelection()?.removeAllRanges();
+}
```

Comment:
I think we can omit this

Reply:
Looks like it's needed.

### With
https://github.com/directus/directus/assets/26413686/3716311d-1626-4085-b3ef-8c497c1860eb

### Without
https://github.com/directus/directus/assets/26413686/bedc7a36-76c9-4891-a168-269cc3f3f2b4

Reply:
Oh, just a Firefox thing then 😅